community/yt-dlp/depends

```diff
@@ -0,0 +1 @@
+python
```

Comment:
This package also requires `zip`, I can't build without it. It doesn't seem to need it on runtime though so you can make it a make dependency.